zephyr/CMakeLists.txt

```diff
@@ -114,14 +114,30 @@ zephyr_include_directories(
 # SOC level sources
 # Files that are commented may not be needed.
 
+
+# New, "de-centralized Zephyr" way. Requires "is_zephyr()" conditionals in
+# the decentralized CMakeLists.txt files shared with XTOS.
+
+# Duplicate in sof/scripts/misc.cmake; keep in sync
+macro(is_zephyr ret)
+	if(CONFIG_ZEPHYR_SOF_MODULE STREQUAL "y")
```

Comment:
Just `if(CONFIG_ZEPHYR_SOF_MODULE)`  should be fine, that's the pattern within Zephyr itself.

With kconfigs, there's a validation layer somewhere (don't ask me where) that guarantees that variables of the form CONFIG_xxx are matched affirmatively with kconfig input.  So you can assume that if such a variable exists that it reflects the same thing you'd see in C or Kconfig and evaluates to false if unset.  Pretty sure, though the details I'm sure are more complicated.

Reply:
Done